google-cloud-spanner/src/test/java/com/google/cloud/spanner/SpannerOptionsTest.java

```diff
@@ -972,12 +982,19 @@ public void checkCreatedInstanceWhenGrpcGcpExtensionDisabled() {
     Spanner spanner2 = options1.getService();
 
     assertNotSame(spanner1, spanner2);
+
+    spanner1.close();
+    spanner2.close();
   }
 
   @Test
   public void checkCreatedInstanceWhenGrpcGcpExtensionEnabled() {
     SpannerOptions options =
-        SpannerOptions.newBuilder().setProjectId("test-project").enableGrpcGcpExtension().build();
+        SpannerOptions.newBuilder()
+            .setProjectId("test-project")
+            .setCredentials(NoCredentials.getInstance())
```

Comment:
This cleans up errors that are being logged during tests, clogging up the test log.